crates/core/tests/integration_test.rs

```diff
@@ -160,7 +160,6 @@ async fn test_host_component_data_update() {
 }
 
 #[tokio::test(flavor = "multi_thread")]
-#[cfg(not(tarpaulin))]
```

Comment:
What was this one about? IIRC this test really did break with tarpaulin...

Reply:
Ah: https://github.com/xd009642/tarpaulin?tab=readme-ov-file#ignoring-code-in-files

> Unfortunately, due to the unexpected cfg warnings cargo now emits you will likely want to add the recommended lints to your Cargo.toml, or utilise any existing build scripts.

Reply:
This seems to be what it means by "utilise any existing build scripts":

https://doc.rust-lang.org/cargo/reference/build-scripts.html#rustc-check-cfg

Anyway, doesn't need to be handled in this PR.